### PR TITLE
[TL-3] Merge updates (too many, next time I do better)

### PR DESCRIPTION
**Pull request by Camel-app**

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        //event.preventDefault();
+        setActive(link.link);
+      }}
+    >
+      {link.label}
+    </a>
+  ));
+
+  return (
+    <footer className={classes.footer}>
```
Comment (Aodarium) [inline components/Footer/Footer.tsx:35]: className={classes.footer} should be deleted

File: components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { Container, Group, Burger, ActionIcon } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import classes from './Footer.module.css';
+import { IconBrandTwitter, IconBrandYoutube } from '@tabler/icons-react';
+
+
+const links = [
+  { link: 'https://github.com/Camel-app', label: 'by Julius Fenn, Florian Gouret', target: "_blank"},
```
Comment (Aodarium) [inline components/Footer/Footer.tsx:9]: Link not valid

File: components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { Container, Group, Burger, ActionIcon } from '@mantine/core';
+import { useDisclosure } from '@mantine/hooks';
+import classes from './Footer.module.css';
+import { IconBrandTwitter, IconBrandYoutube } from '@tabler/icons-react';
```
Comment (Aodarium) [inline components/Footer/Footer.tsx:5]: Icon not used

File: components/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+export function FooterSimple() {
+  const [opened, { toggle }] = useDisclosure(false);
+  const [active, setActive] = useState(links[0].link);
+
+  const items = links.map((link) => (
+    <a
+      key={link.label}
+      href={link.link}
+      target={link.target}
+      className={classes.link}
```
Comment (Aodarium) [inline components/Footer/Footer.tsx:23]: className={classes.footer} should be deleted

File: components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        setActive(link.link);
+      }}
+    >
+      {link.label}
+    </a>
+  ));
+
+  return (
+    <footer className={classes.footer}>
+      <Container size="md" className={classes.inner}>
```
Comment (Aodarium) [inline components/Footer/Footer.tsx:36]: className={classes.footer} should be deleted

File: components/Footer/Footer.module.css [hunk 6]
```diff
@@ -0,0 +1,36 @@
+.footer {
```
Comment (Aodarium) [inline components/Footer/Footer.module.css:1]: See if there is a component to copy past here from mantine

File: components/Information/Information.tsx [hunk 7]
```diff
@@ -0,0 +1,255 @@
+import {
+  Title,
+  Text,
+  Anchor,
```
Comment (Aodarium) [inline components/Information/Information.tsx:4]: Some imports not used

File: components/Information/Information.tsx [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+import { IconArrowAutofitRight, IconHelp, IconPhoneCalling, IconUsers } from '@tabler/icons-react';
+
+import image from '../../public/logoCAM2.svg';
+
+import classes from './Information.module.css';
+
+export function Information() {
+  return (
+    <>
+      <Title className={classes.title} ta="center" mt={10} mb={20}>
```
Comment (Aodarium) [inline components/Information/Information.tsx:22]: className={classes.footer} should be deleted

File: components/Information/Information.tsx [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+        <Text
+          inherit
+          variant="gradient"
+          component="span"
+          gradient={{ from: 'blue', to: 'darkblue' }}
+        >
+          Thank you for checking out our Cognitive Affective Map tools!
+        </Text>
+      </Title>
+      <Title className={classes.subtitle} ta="center">
```
Comment (Aodarium) [inline components/Information/Information.tsx:32]: className={classes.footer} should be deleted

File: components/Information/Information.tsx [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+          <a
+            key="Documentation"
+            href="https://camtools-documentation.readthedocs.io/en/master/Set%20up%20study/"
+            target="_blank"
+          >
+            <span>Documentation</span>
+          </a>
+        </Grid.Col>
+      </Grid>
+      <Title className={classes.subtitle} ta="center">
```
Comment (Aodarium) [inline components/Information/Information.tsx:72]: className={classes.footer} should be deleted

File: components/Information/Information.tsx [hunk 11]
```diff
@@ -0,0 +1,255 @@
...
+            key="CAM-App documentation"
+            href="https://camtools-documentation.readthedocs.io/en/master/CAM-App/"
+            target="_blank"
+          >
+            <span>CAM-App documentation</span>
+          </a>
+          ).
+        </List.Item>
+      </List>
+      <Title className={classes.subtitle} ta="center" mt={40}>
```
Comment (Aodarium) [inline components/Information/Information.tsx:125]: className={classes.footer} should be deleted

File: components/Information/Information.tsx [hunk 12]
```diff
@@ -0,0 +1,255 @@
...
+          Set up the C.A.M.E.L. software on your own server to have full control. Here it is
+          possible to implement even more advanced features (for inspiration see future features
+          sections in the documentation).
+        </List.Item>
+        <List.Item>
+          If you want to extend the features of the CAM-App you can freely adjust our written
+          functions in R or write your own (check out our source code).
+        </List.Item>
+      </List>
+      <Title className={classes.subtitle} ta="center" mt={40}>
```
Comment (Aodarium) [inline components/Information/Information.tsx:157]: className={classes.footer} should be deleted

File: components/Information/Information.tsx [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+  Group,
+  Center,
+  Image,
+  Grid,
+} from '@mantine/core';
+import { IconArrowAutofitRight, IconHelp, IconPhoneCalling, IconUsers } from '@tabler/icons-react';
+
+import image from '../../public/logoCAM2.svg';
+
+import classes from './Information.module.css';
```
Comment (Aodarium) [inline components/Information/Information.tsx:17]: Not sure if needed here

File: components/Information/Information.tsx [hunk 14]
```diff
@@ -0,0 +1,255 @@
+import {
```
Comment (Aodarium) [inline components/Information/Information.tsx:1]: Layout of the page not working

File: components/Welcome/Welcome.tsx [hunk 15]
```diff
@@ -1,23 +1,65 @@
-import { Title, Text, Anchor } from '@mantine/core';
+import { Title, Text, Anchor, List, ThemeIcon, Button, Group, Center } from '@mantine/core';
```
Comment (Aodarium) [inline components/Welcome/Welcome.tsx:1]: delete all className ref

Layout not working

File: components/Welcome/Welcome.tsx [hunk 16]
```diff
@@ -1,23 +1,65 @@
...
+        <Text
+          inherit
+          variant="gradient"
+          component="span"
+          gradient={{ from: 'blue', to: 'darkblue' }}
+        >
+          Cognitive Affective Map tools to easily create and integrate CAM studies into online
+          experiments.
+        </Text>
+        <Text color="dimmed" mt="md">
```
Comment (Aodarium) [inline components/Welcome/Welcome.tsx:20]: Color is not valid anymore

File: components/Welcome/Welcome.tsx [hunk 17]
```diff
@@ -1,23 +1,65 @@
...
+        </List.Item>
+      </List>
+      <Center>
+        <Group mt={30} ta="center">
+          <a key="More information" href="/information" target="_self">
+            <Button radius="xl" size="md">
+              More information
+            </Button>
+          </a>
+          <a key="Source code" href="https://github.com/Camel-app" target="_blank">
```
Comment (Aodarium) [inline components/Welcome/Welcome.tsx:56]: Wrong link

**Review by Aodarium — CHANGES_REQUESTED** [review]

File: components/Information/Information.tsx (hunk 13 above)
Comment (Camel-app) [reply components/Information/Information.tsx:17]: I will check this!

File: components/Welcome/Welcome.tsx (hunk 15 above)
Comment (Camel-app) [reply components/Welcome/Welcome.tsx:1]: I reformat it and only included classname for title / subtitles

File: components/Welcome/Welcome.tsx (hunk 16 above)
Comment (Camel-app) [reply components/Welcome/Welcome.tsx:20]: instead it is simply "c"?

File: components/Footer/Footer.tsx (hunk 2 above)
Comment (FennStatistics) [reply components/Footer/Footer.tsx]: Why?

Comment (FennStatistics) [reply components/Footer/Footer.tsx]: is removed

File: components/Footer/Footer.tsx (hunk 3 above)
Comment (FennStatistics) [reply components/Footer/Footer.tsx]: is removed

File: components/Footer/Footer.tsx (hunk 1 above)
Comment (FennStatistics) [reply components/Footer/Footer.tsx]: done

**Pull request closed without merging**